frontend/src/expert/components/SidePanelCard.js

```diff
@@ -226,7 +226,7 @@ function SidePanelCard(props) {
         <Card className='scroll panel'>
             {console.log('rendering sidepanelcard')}
             <Card.Body>
-                <Card.Title style={{ color: 'white' }}>WS Functions</Card.Title>
+                <Card.Title style={{ color: 'white' }}>Auto-fill Functions</Card.Title>
```

Comment:
How about just "functions"?

Reply:
Perhaps "Built-In Functions" since the custom functions are added to the top, above the "builtins"?

Reply:
Sure, that sounds good!